src/pages/academy/groundControl/GroundControlContainer.ts

```diff
@@ -23,6 +24,7 @@ const mapDispatchToProps: MapDispatchToProps<DispatchProps, {}> = (dispatch: Dis
   bindActionCreators(
     {
       handleAssessmentChangeDate: changeDateAssessment,
+      // handleAssessmentChangeTeamSize: changeTeamSizeAssessment,
```

Comment:
Commented code – what's this for? Is it a future feature?

Reply:
Removed.